### DISCUSSION TITLE
PASSWALL首页上的百度连接PING 数值为什么会高一倍？

### DISCUSSION THREAD

**Discussion by etsam** [Openwrt-Passwall/openwrt-passwall2#452] [Q&A] [answered]
![011](https://github.com/xiaorouji/openwrt-passwall2/assets/24689501/619ebf61-c5fc-4665-bdff-95accab02fb2)
![022](https://github.com/xiaorouji/openwrt-passwall2/assets/24689501/2a577e83-a3d3-4ed9-a434-3efdba82a01c)
![033](https://github.com/xiaorouji/openwrt-passwall2/assets/24689501/8bf0868d-157e-4226-be4c-aa9fe38b9a5f)

如图所示，在电脑本机直接ping www.baidu.com 和在路由器上ping 数值基本一致（24-27ms）但是在PW2首页里， 不管是代理开启或者不开启，ping值都差不多高一倍(42-55ms)。这个问题一直困扰我，不知道是哪里出了问题，虽然这个问题不影响使用，但是我想搞明白，无奈没有数据对比。
请问大家是否也有同样的PING数值差异。如果没有，我就自己在找找原因，谢谢!

**Comment by leslieqwq** [comment]
因为https握手次数多了吧，不像你icmp ping

**Comment by Travel2Here** [comment, marked as answer]
ping http https 分别是不同的协议 耗时不同，目前显示的数值是http协议的，一般比ping高一倍，而https则可能是3~4倍。
结论：你的描述现象一切正常 可以不服用焦虑安眠药

**Comment by etsam** [comment]
我当时也是猜测可能是这个原因， 直到看到一个UP主上传视频  看到他百度PING只有14，所以我才怀疑自己的是不是在什么地方设置出了问题。按高一倍的设定 这人链接百度的ping值这么低 难道在百度机房。 

![1111](https://github.com/xiaorouji/openwrt-passwall2/assets/24689501/cbb79d18-1a68-473b-9c29-6e5784f8ec41)

谢谢你们的解惑，不纠结了！^_^

**Reply by Travel2Here** [reply]
同工具同协议下的数值区别，只能是各自网络条件（线路质量、机房节点距离等等）、以及设备转发能力（包括软硬件，软的方面可能你们的内核不一样）的差异了。
关系不大 只要你确定baidu走的直连就已经到天花板了~
